src/ResourceManager/SubscriptionDefinition/AzureRM.SubscriptionDefinition.psd1

```diff
@@ -0,0 +1,126 @@
+﻿#
+# Module manifest for module 'PSGet_AzureRM.SubscriptionDefinition'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 8/7/2017
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.1.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'bb985428-37c8-42cc-9193-58523906bd6d'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - Subscription Definition cmdlets for Azure Resource Manager'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '3.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.5.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '3.3.2'; })
+
+# Assemblies that must be loaded prior to importing this module
+# RequiredAssemblies = @()
```

Comment:
@wilcobmsft uncomment this field and make sure to include any dlls that your project depends on (that is not found in [`Common.Dependencies.targets`](https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets). It looks it should just be `Microsoft.Azure.Management.ResourceManager`

Reply:
Fixed.